src/Components/BatchStatusList.test.tsx

```diff
@@ -0,0 +1,159 @@
+            </Router>
+        );
+        
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("Check run table:", () => {
+    it("displays table headings (including loader/spinner)", async () => {
+        const history = createMemoryHistory({
+            initialEntries: [mockRoute]
+        });
+        render(
+            <Router history={history}>
+                <BatchStatusList statusDescriptionList={statusDescriptions} />
+            </Router>
+        );
+
+        expect(screen.getByText(/Loading/i)).toBeInTheDocument();
+        expect(await screen.findByText(/Questionnaire/)).toBeVisible();
+        expect(await screen.findByText(/Status/)).toBeVisible();
+        expect(await screen.findByText(/Last update/)).toBeVisible();
+        expect(screen.queryByText(/Loading/i)).not.toBeInTheDocument();
+    });
+
+    it("displays the questionnaire/instrument names", async () => {
+        const history = createMemoryHistory({
+            initialEntries: [mockRoute]
+        });
+        render(
+            <Router history={history}>
+                <BatchStatusList statusDescriptionList={statusDescriptions} />
+            </Router>
+        );
+
+        expect(await screen.findByText(/OPN2004A/)).toBeVisible();
+        expect(await screen.findByText(/OPN2101A/)).toBeVisible();
+        expect(await screen.findByText(/OPN2331A/)).toBeVisible();
+        expect(await screen.findByText(/OPN2806A/)).toBeVisible();
+    });
+
+    it("displays the status colour and details", async () => {
+        const history = createMemoryHistory({
+            initialEntries: [mockRoute]
+        });
+        render(
+            <Router history={history}>
+                <BatchStatusList statusDescriptionList={statusDescriptions} />
+            </Router>
+        );
+
+        expect(await screen.findByText(statusDescriptions["in_arc"])).toBeVisible();
+        expect(await screen.findByTestId(/OPN2004A-status--success/)).toBeDefined();
+        expect(await screen.findByText(statusDescriptions["generated"])).toBeVisible();
+        expect(await screen.findByTestId(/OPN2101A-status--pending/)).toBeDefined();
+        expect(await screen.findByText(statusDescriptions["inactive"])).toBeVisible();
+        expect(await screen.findByTestId(/OPN2331A-status--dead/)).toBeDefined();
+        expect(await screen.findByText(statusDescriptions["errored"])).toBeVisible();
+        expect(await screen.findByTestId(/OPN2806A-status--error/)).toBeDefined();
+    });
+
+    it("displays last update details", async () => {
+        const history = createMemoryHistory({
+            initialEntries: [mockRoute]
+        });
+        render(
+            <Router history={history}>
+                <BatchStatusList statusDescriptionList={statusDescriptions} />
+            </Router>
+        );
+        
+        expect(await screen.findByText(dateFormatter(batchRuns[0].updated_at).format("DD/MM/YYYY HH:mm:ss"))); // 24/03/2021 12:21:10
```

Comment:
An expect should have something after it (e.g. `expect(...).toSomthing()`) - we probably want `.toBeVisible()` here?